README.md

```diff
@@ -8,6 +8,19 @@ Multiple scenarios will be addressed:
 3. Increasing penetration of variable renewable energy sources.
 4. Add grid flexibility in the form of H2 and thermal storage.
 
+## Instructions to Run TEMOA
+TEMOA is an open source modeling tool available on GitHub (link). Follow the installation instructions here (link). 
+
+After creating a database in sql, navigate to the directory with your database:
+
+``sqlite3 [filename].sqlite < [filename].sql``
+
+if you don't have sqlite installed, run:
+
+``sudo apt-get install sqlite``
```

Comment:
not `sudo apt-get install sqlite3` ?

Reply:
I did ``sqlite`` and run ``sqlite3`` just fine. I'm not sure if there is a difference or it's sort of like specifying ``python`` vs ``python3`` (pre-2020, that is).